### DISCUSSION TITLE
Support for temporal data

### DISCUSSION THREAD

**Discussion by pierre0042** [mydumper/mydumper#895] [Q&A] [answered]
Hello, 

I would like to know if mydumper supports the management of temporal data in backups, for example System-Versioned tables?

Thanks

**Comment by davidducos** [comment, marked as answer]
Hi @pierre0042 , 

Not sure what do you mean with "temporal data in backups", if it is just https://mariadb.com/kb/en/system-versioned-tables/, I can answer that we are not currently supporting System-Versioned tables. 

We might do it in the future, I'm wondering about your expectations about it and the use cases that you have in mind. Can you elaborate further?

**Comment by pierre0042** [comment]
Hi David,

Thanks for your answer, this is what I was referring to.
I have a database using this functionality, so I need to be able to backup it with the versioning data as well.

**Comment by LinuxJedi** [comment]
Hi @pierre0042,

This can be done as a future feature, but with the caveat that it would only work against MariaDB 10.11. Before that at the very least you would not be able to restore the temporal data. I'd have to look into whether or not the backup is possible in older versions.
